container/check_container.go

```diff
@@ -38,8 +38,41 @@ func NewCheck(image string, opts ...Option) *containerCheck {
 // pyxis information is provided. Calls should add a relevant ArtifactWriter to the context if they
 // wish to work with artifact files written by checks.
 func (c *containerCheck) Run(ctx context.Context) (certification.Results, error) {
+	pol, checks, err := c.List(ctx)
+	if err != nil {
+		return certification.Results{}, fmt.Errorf("%w: %s", preflighterr.ErrCannotListChecks, err)
```

Comment:
This will fix the failing tests

```suggestion
		return certification.Results{}, err
```

Reply:
fixed